src/studio-header/header.scss

```diff
@@ -70,7 +70,6 @@ $white: #fff;
 .site-header-desktop {
   height: 3.75rem;
   box-shadow: 0 1px 0 0 rgba(0, 0, 0, 0.1);
-  background: $white;
```

Comment:
Should be remove the scss variable declarations on top of the file ? 

```
$spacer: 1rem;
$blue: #007db8;
$white: #fff;
```
For some reason I am not able to comment on the lines at the top of the file, so leaving the comments here.

Reply:
We don't need to since they are local to this MFE. The only issue is with using variables from paragon.